src/pages/Delegate/MultiTransactionDialog.tsx

```diff
@@ -108,39 +112,33 @@ export const MultiTransactionDialog = ({
       return
     }
 
-    const subscriptionCallBack2 = getSubscriptionCallBack({
+    onSignDelegations({
       onError: () => {
         setIsTxInitiated(false)
       },
       onInBlock: () => {
-        onProcessFinished()
         setIsTxInitiated(false)
       },
     })
-
-    await step2Txs
-      .signSubmitAndWatch(selectedAccount?.polkadotSigner, { at: 'best' })
-      .subscribe(subscriptionCallBack2)
   }, [
     api,
     delegateTxs.delegationTxs,
-    getSubscriptionCallBack,
     isExhaustsResources,
-    onProcessFinished,
+    onSignDelegations,
     selectedAccount,
   ])
 
   if (promptForHelpCallData)
     return (
       <TooLargeDialog
-        isOpen={isOpen}
+        isOpen
         onOpenChange={onOpenChange}
         callData={promptForHelpCallData}
       />
     )
 
   return (
-    <Dialog open={isOpen} onOpenChange={onOpenChange}>
+    <Dialog open onOpenChange={onOpenChange}>
```

Comment:
This should be `isOpen` - no?

```suggestion
    <Dialog isOpen onOpenChange={onOpenChange}>
```

Reply:
no, you can see the previous was `open={isOpen}`, leaving just `open` means it'll be `true` always, which is what I want to achieve. The opening is managed differently here.